src/architecture/serialization.md

```diff
@@ -474,22 +494,20 @@ let expected_buffer_str = pub_key_str.to_owned() + // Public key
                           "06000000" +             // Segment size
                           "d204000000000000" +     // Balance
                           "416e64726577";          // Name
-
-let expected_buffer: Vec<u8> = HexValue::from_hex(&expected_buffer_str)
-                                       .unwrap();
-
+let expected_buffer = Vec::<u8>::from_hex(&expected_buffer_str)
+    .unwrap();
 assert_eq!(my_wallet.serialize(), expected_buffer);
 ```
 
 Serialized representation of `my_wallet`:
 
 | Position | Stored data  | Hexadecimal form | Comment |
 |:--------|:------:|:---------------------|:--------------------------------------------------|
-`0 => 32`  |       | `99 ac e6 c7 21 db 29 3b 0e d5 b4 87 e6 d6 11 1f 22 a8 c5 5d 2a 1b 76 06 b6 fa 6e 6c 29 67 1a a1` | Public key |
-`32  => 36`  | 48    | `30 00 00 00`            | Little endian stored segment pointer, refer to position in data where real string is located |
-`36  => 40`  | 6     | `06 00 00 00`            | Little endian stored segment size |
-`40  => 48` | 1234   | `d2 04 00 00 00 00 00 00`| Number in little endian |
-`48 => 54` | Andrew| `41 6e 64 72 65 77`       | Text bytes in UTF-8 encoding |
+0..32  |       | `99 ac e6 c7 21 db 29 3b 0e d5 b4 87 e6 d6 11 1f 22 a8 c5 5d 2a 1b 76 06 b6 fa 6e 6c 29 67 1a a1` | Public key |
+32..36  | 48    | `30 00 00 00`            | Little endian stored segment pointer, refer to position in data where real string is located |
```

Comment:
rephrase as "A little endian segment pointer that refers to the string position in the serialization buffer"